python/templates/Collection.jl.jinja2

```diff
@@ -0,0 +1,2 @@
+include("{{ class.bare_type }}.jl")
+{{ class.bare_type }}Collection = Vector{ {{ class.bare_type }}{% if params_jl %}{ {% for relation in params_jl %} {{ relation }}, {% endfor %} }{% endif %} }
```

Comment:
```suggestion
{{ class.bare_type }}Collection = Vector{ {{ class.bare_type }}{% if params_jl %}{ {{ params_jl | join(', ') }} }{% endif %} }
```
For consistency with how it is done in the constructors

Reply:
`Collection.jl.jinja2` was a mistake.